recipes-bsp/bootfiles/rpi-config_git.bb

```diff
@@ -225,6 +225,12 @@ do_deploy() {
         echo "# Enable USB host mode" >> $CONFIG
         echo "dtoverlay=dwc2,dr_mode=host" >> $CONFIG
     fi
+    
+    # DWC2 USB OTG support
+    if [ "${ENABLE_DWC2_OTG}" = "1" ] && [ "${ENABLE_DWC2_PERIPHERAL}" != "1" ]; then
```

Comment:
does this work with non-bash shells ?

Reply:
To the best of my knowledge (please correct me if I am wrong), but Yocto wants us to work with dash, where this logic does work. The syntax for if-else changes for csh or tcsh, so this scripting will not work there.

Reply:
Yes that should be enough